encord/orm/client_metadata_schema.py

```diff
@@ -0,0 +1,38 @@
+import uuid
+from typing import Dict, Type, Any
+from uuid import UUID
+
+from encord.common.enum import StringEnum
+from encord.orm.base_dto import BaseDTO
+from encord.orm.base_dto.base_dto_interface import T
+
+
+class ClientMetadataSchemaTypes(StringEnum):
+    NUMBER = "number"
+    STRING = "string"
+    BOOLEAN = "boolean"
+    DATETIME = "datetime"
+    GEOSPATIAL = "geospatial"
+
+
+class ClientMetadataSchema(BaseDTO):
+    uuid: UUID
+    metadata_schema: Dict[str, ClientMetadataSchemaTypes]
+    organisation_id: int
+
+    @classmethod
+    def from_dict(cls: Type[T], d: Dict[str, Any]) -> T | None:
```

Comment:
I wonder, can it be replaced with standard pydantic `Field` to set defaults rather than having a method? 
probably would be more straightforward?